libr/main/radare2.c

```diff
@@ -1506,8 +1506,6 @@ R_API int r_main_radare2(int argc, const char **argv) {
 	free (envprofile);
 	free (debugbackend);
 	r_core_free (r);
-	r_cons_set_raw (0);
```

Comment:
umf why remove those lines?

Reply:
Because https://github.com/radareorg/radare2/blob/8245e459f48ba89ba9135115de4fa8e666bcad5d/libr/core/core.c#L3043 r_core_free already called r_cons_free. Another call will make r_cons_instance.refcnt -1.